opentracing-cdi-example/src/main/java/io/opentracing/contrib/cdi/example/OrderPlacementEvent.java

```diff
@@ -0,0 +1,15 @@
+package io.opentracing.contrib.cdi.example;
+
+import io.opentracing.SpanContext;
+
+public class OrderPlacementEvent {
+    private SpanContext spanContext;
+
+    public OrderPlacementEvent(SpanContext spanContext) {
```

Comment:
This is actually following Google's pattern of explicit context propagation (also used on Dapper, IIRC), but users aren't required to do so. See the other event, `InventoryChangeEvent`, for a no-context example.